doc/getting_started_linux.rst

```diff
@@ -23,109 +24,48 @@ Optional dependencies (for development and documentation of the p-net stack)::
 
     sudo apt install -y \
         clang-tools \
+        clang-format-10 \
         doxygen \
         graphviz
 
 
-Download and compile p-net
----------------------------
-Clone the source::
+OSAL details
```

Comment:
I am not sure this level of detail is required here, in particular because this will be the common behaviour among all ports and projects using OSAL. This text for instance applies to rt-kernel also.

It would be better to document the detailed behaviour in the OSAL project and link to that text.

Reply:
Great idea. 
In the mean time I added a reference in `doc/getting_started_rtkernel.rst`
Done